code/modules/medical/genetics/bioEffects/non_genetic.dm

```diff
@@ -271,3 +271,130 @@
 			overlay_image = image("icon" = 'icons/effects/genetics.dmi', "icon_state" = "outline", layer = MOB_LIMB_LAYER)
 			overlay_image.color = "#007BFF"
 		..()
+
+////////////////////
+//Wraith curses
+////////////////////
+
+/datum/bioEffect/blood_curse
+	name = "Blood curse"
+	desc = "Curse of blood."
+	id = "blood_curse"
+	effectType = EFFECT_TYPE_DISABILITY
+	can_copy = 0
+	isBad = 1
+	occur_in_genepools = 0
+	acceptable_in_mutini = 0
+	probability = 0
+	curable_by_mutadone = 0
+
+	OnAdd()
+		if (ishuman(owner))
+			owner.traitHolder?.addTrait("hemophilia")
+
+	OnLife()
+		var/mult = 1
```

Comment:
i like how mult is just always 1

just avoiding the actual problem lol

Reply:
What makes this odd to me is that OnLife() actually gets passed a mult var already